fe/fe-core/src/main/java/com/starrocks/sql/optimizer/cost/CostModel.java

```diff
@@ -262,7 +262,8 @@ public CostEstimate visitPhysicalHashJoin(PhysicalHashJoinOperator join, Express
                     "should be handled by nestloopjoin");
 
             return CostEstimate.of(leftStatistics.getOutputSize(context.getChildOutputColumns(0))
-                            + rightStatistics.getOutputSize(context.getChildOutputColumns(1)),
+                            + rightStatistics.getOutputSizeIncreaseRowWeight(context.getChildOutputColumns(1),
+                            ConnectContext.get().getSessionVariable().getCboJoinCostRowWeight()),
```

Comment:
ContextContext.get() may be nullable here, so I think it is better to check wether or not you can get the right session variable before using them.

Reply:
> ContextContext.get() may be nullable here, so I think it is better to check wether or not you can get the right session variable before using them.

Thanks a lot, i have fix this.